core-bundle/contao/library/Contao/Model.php

```diff
@@ -391,10 +391,18 @@ public function mergeRow(array $arrData)
 	 *
 	 * @return array<string,array<string,string>>
 	 */
-	public static function getColumnCastTypes(): array
+	public static function getColumnCastTypes(bool $fromDatabase): array
```

Comment:
```suggestion
	public static function getColumnCastTypes(bool $fromDatabase = false): array
```

Since it is a public static function, we should probably not enforce the argument.

Reply:
It is marked as `@internal`, so this should be fine IMO.

Reply:
Ah, I missed that. I still think a default argument would be nicer, though. 🙈

Reply:
Split into two methods in e5104da699d0105ecdb1ba9baaadde5096583b3a